tests/testthat/test-calculateNotApplicableStatus.R

```diff
@@ -0,0 +1,87 @@
+library(testthat)
+
+test_that("Not Applicable status Table Empty", {
+  outputFolder <- tempfile("dqd_")
+  on.exit(unlink(outputFolder, recursive = TRUE))
+
+  results <- executeDqChecks(
+    connectionDetails = connectionDetailsEunomia,
+    cdmDatabaseSchema = cdmDatabaseSchemaEunomia,
+    resultsDatabaseSchema = resultsDatabaseSchemaEunomia,
+    cdmSourceName = "Eunomia",
+    checkNames = c("cdmTable", "cdmField", "measureValueCompleteness"),
+    # Eunomia COST table has misspelled 'REVEUE_CODE_SOURCE_VALUE'
+    tablesToExclude = c("COST", "CONCEPT", "VOCABULARY", "CONCEPT_ANCESTOR", "CONCEPT_RELATIONSHIP", "CONCEPT_CLASS", "CONCEPT_SYNONYM", "RELATIONSHIP", "DOMAIN"),
+    outputFolder = outputFolder,
+    writeToTable = F
+  )
+
+  # Assumption that Eunomia has empty device_exposure table
```

Comment:
a bold move ;) if eunomia updates to fill that table, then our test starts failing. maybe clear it out like you did below, to be safe?